itest.py

```diff
@@ -0,0 +1,832 @@
+#! /usr/bin/env python3
+
+import logging
+from logging.handlers import SysLogHandler
+import os
+import subprocess
+import sys
+import shutil
+from pathlib import PurePath, Path
+import docker
+from docker.types import Mount
+import re
+import jsonschema
+import argparse
+import json
+
+
+# ------------
+# LOGGER STUFF
+# ------------
+logger = logging.getLogger('irods-test-runner')
+logger.setLevel(logging.DEBUG)
+
+# Console logging
+ch = logging.StreamHandler()
+ch.setLevel(logging.DEBUG)
+
+# Syslog logging
+sh = SysLogHandler('/dev/log')
+sh.setLevel(logging.DEBUG)
+
+formatter = logging.Formatter('[%(asctime)s %(name)s %(levelname)s] %(message)s')
+ch.setFormatter(formatter)
+
+sys_formatter = logging.Formatter('[thread=%(thread)d name=%(name)s] %(message)s')
+sh.setFormatter(sys_formatter)
+
+logger.addHandler(ch)
+logger.addHandler(sh)
```

Comment:
The logging should be simplified. Most of this was experimenting with different types of logging.

Reply:
Removed syslog logging, leaving console logging for now.